### PR TITLE
new updated extension

### PR DESCRIPTION
**Pull request by ramuksubam**

### CODE REVIEW

File: samples/extensions/AWSSQS/activity/sqsreceivemessage/activity.json [hunk 1]
```diff
@@ -88,4 +106,4 @@
 			"type": "complex_object"
```
Comment (vijaynalawade) [inline samples/extensions/AWSSQS/activity/sqsreceivemessage/activity.json]: Should be object type?

File: samples/extensions/AWSSQS/activity/sqssendmessage/activity.json [hunk 2]
```diff
@@ -1,72 +1,79 @@
...
+        "name": "Message Attributes",
+        "description": "Set message attributes",
+        "type": "table",
+        "schema": "{\r\n    \"$schema\": \"http:\/\/json-schema.org\/draft-04\/schema#\",\r\n    \"definitions\": {},\r\n    \"id\": \"http:\/\/example.com\/example.json\",\r\n    \"items\": {\r\n        \"id\": \"\/items\",\r\n        \"properties\": {\r\n            \"Name\": {\r\n                \"id\": \"\/items\/properties\/Name\",\r\n                \"type\": \"string\"\r\n            },\r\n            \"Type\": {\r\n                \"id\": \"\/items\/properties\/Type\",\r\n                \"type\": {\"enum\":[\"String\", \"Number\"]}\r\n            }\r\n        },\r\n        \"type\": \"object\"\r\n    },\r\n    \"type\": \"array\"\r\n}",
+        "inputSettingsOnly":true
+      }
+    },
+    {
+      "name": "MessageAttributes",
+      "type": "complex_object",
```
Comment (vijaynalawade) [inline samples/extensions/AWSSQS/activity/sqssendmessage/activity.json]: should be object type

File: samples/extensions/AWSSQS/activity/sqssendmessage/activity.ts [hunk 3]
```diff
@@ -59,46 +59,47 @@ export class SendMsgActivityContribution extends WiServiceHandlerContribution {
                         jsonSchema[data[i].Name] = 0;
                     }
                 }
+                console.log(JSON.stringify(jsonSchema));
```
Comment (vijaynalawade) [inline samples/extensions/AWSSQS/activity/sqssendmessage/activity.ts]: Remove console log

File: samples/extensions/AWSSQS/connector/sqs/connector.json [hunk 4]
```diff
@@ -45,8 +46,8 @@
       "type": "string",
       "required": true,
       "display": {
-        "name": "Secret Access Key",
-        "description": "AWS Secret Access Key for the user",
+        "name": "Secrete Access Key",
```
Comment (vijaynalawade) [inline samples/extensions/AWSSQS/connector/sqs/connector.json]: Should be **Secret**

File: samples/extensions/AWSSQS/connector/sqs/connector.json [hunk 5]
```diff
@@ -45,8 +46,8 @@
...
+        "description": "AWS Secrete Access Key for the user",
```
Comment (vijaynalawade) [inline samples/extensions/AWSSQS/connector/sqs/connector.json]: Should be **Secret**

File: samples/extensions/AWSSQS/activity/sqssendmessage/activity.ts (hunk 3 above)
Comment (ramuksubam) [reply samples/extensions/AWSSQS/activity/sqssendmessage/activity.ts]: done with the changes

File: samples/extensions/AWSSQS/connector/sqs/connector.json (hunk 4 above)
Comment (ramuksubam) [reply samples/extensions/AWSSQS/connector/sqs/connector.json]: have made the changes and also have made the connector.ts file changes.

File: samples/extensions/AWSSQS/connector/sqs/connector.json (hunk 5 above)
Comment (ramuksubam) [reply samples/extensions/AWSSQS/connector/sqs/connector.json]: done with this changes as well

**Comment by ramuksubam** [comment]
Can i know what we should write in the attributes settings description for `sqssendmessage` and `sqsreceiveactivity`

**Comment by lixingwang** [comment]
@vijaynalawade @ramuksubam :  Should we consider backward compatible?  We move the connection to settings which will break backward compatible.

**Pull request merged**